src/reducers/clusterReducer.js

```diff
@@ -179,78 +134,6 @@ export default function clusterReducer(
         items: items,
       };
 
-    case types.CLUSTER_LOAD_METRICS:
```

Comment:
This and the other removals here was dead code, the newer version of babel detected this and raised errors about it.